legend-engine-core/legend-engine-core-pure/legend-engine-pure-code-compiled-core/src/main/resources/core/pure/router/routing/router_routing.pure

```diff
@@ -784,6 +784,8 @@ function meta::pure::router::routing::shouldStopFunctions(extensions:meta::pure:
     left_String_1__Integer_1__String_1_,
     right_String_1__Integer_1__String_1_,
     typeName_Any_1__String_1_,
+    levenshteinDistance_String_1__String_1__Integer_1_,
```

Comment:
Are these needed? Native functions are anyways stop functions

Reply:
Didn't know, and I see other native functions like lpad, rpad as well. Since it doesn't impact anything will leave it there.